lusidtools/apps/flush_transactions.py

```diff
@@ -0,0 +1,211 @@
+import logging
+import lusid
+import pathlib
+from lusidtools.lpt import stdargs
+from lusidtools.logger import LusidLogger
+
+
+def parse(extend=None, args=None):
+    return (
+        stdargs.Parser("Flush Transactions", ["scope", "portfolio", "date_range",],)
+        .extend(extend)
+        .parse(args)
+    )
+
+
+def transaction_batcher_by_character_count(
+    scope: str, code: str, host: str, input_lst: list, maxCharacterCount: int = 4000
+):
+    """
+    Takes a given input list of transactions or transactionIDs and batches them based on a maximum number of
+    characters allowed in the URI
+
+    Parameters
+    ----------
+    scope : str
+        Scope of the portfolio in question
+    code : str
+        Code of the portfolio in question
+    host : str
+        Host taken from the configuration of the api client. Represents the start the base url involved in the calls
+    input_lst : list
+        A list containing all the transaction information
+    maxCharacterCount : int
+        Maximum number of characters allowed in the URI
+
+    Returns
+    -------
+    batched_data : list of lists
+        Returns a list with each item being a batch (which is itself, a list of transactions)
+
+    """
+
+    # Determine how many characters are fixed to be included in url before the addition of query params
```

Comment:
Can you get this from the SDK instead of hardcoding it?

Reply:
It can't be taken from the SDK as the SDK isn't what is limiting it, the failure from a too long URI comes as an API response rather than some for of data validation